app/controllers/api/v1/subject_sets_controller.rb

```diff
@@ -3,8 +3,32 @@ class Api::V1::SubjectSetsController < Api::ApiController
   resource_actions :default
   schema_type :json_schema
 
+  def create
+    super { |workflow| refresh_queue(workflow) }
```

Comment:
i think the block param should be called subjet_set for this controller, more changes below too.